content/blogposts/2021/postgres-version-update.md

```diff
@@ -99,21 +99,25 @@ The **approach we chose** tracks the minimum and maximum row versions for each i
 
 _lsif_data_definitions_:
 
+<TableWrapper>
 | index_id | scheme | identifier | data | version |
 | -------- | ------ | ---------- | ---- | ------- |
 | 1        | foo    | bar        | .... | 1       |
 | 1        | bar    | baz        | .... | 2       |
 | 2        | baz    | quux       | .... | 2       |
 | 2        | quux   | bonk       | .... | 2       |
 | 3        | bonk   | honk       | .... | 1       |
+</TableWrapper>
```

Comment:
Nice! Was curious how markdown written tables would work with the wrapper 🤓

Reply:
Isn't it unexpected? I was concerned we would need to rewrite them in html, but it looks like it gets parsed in time to be read with the wrapper. I would prefer to use one standard, but for UX I think we should support an author's preference. Markdown tables are also more intuitive for humans to read 🤖

Reply:
Come to think of it, [running through the steps](https://en.wikipedia.org/wiki/Rubber_duck_debugging) makes sense since everything is pre-rendered for the SSG build. Neato! 🤓 

That's a great point about markdown tables being easier to read and I agree; giving the user their preference of writing style is a big positive! 😄 Solid work Timski!

Reply:
You've got it, Brett! (TIL I need a desktop duck.)